example-deployment/run-reporting-sever-migrations.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
parts of this seems to be duplicated in `deploy.sh`

Reply:
Removed this, because it is useless after having the conversations above